src/components/ModelsPage/modelsPage.util.js

```diff
@@ -17,12 +17,50 @@ illegal under applicable law, and the grant of the foregoing license
 under the Apache 2.0 license is conditioned upon your compliance with
 such restriction.
 */
-import React from 'react'
+import React, { useCallback, useState } from 'react'
+import { useDispatch } from 'react-redux'
+import { useParams } from 'react-router-dom'
+import { useYaml } from '../../hooks/yaml.hook'
+
+import artifactsAction from '../../actions/artifacts'
```

Comment:
empty line should be between `useParams` and `useYaml` instead of `useYaml` and `artifactsAction`